cmake/pcl_find_vtk.cmake

```diff
@@ -75,16 +75,8 @@ set(NON_PREFIX_PCL_VTK_COMPONENTS
   ViewsContext2D
 )
 
-#If VTK version 6 use OpenGL
-if(VTK_VERSION VERSION_LESS 7.0)
-  set(VTK_RENDERING_BACKEND "OpenGL")
-  set(VTK_RENDERING_BACKEND_OPENGL_VERSION "1")
-  message(DEPRECATION "The rendering backend OpenGL is deprecated and not available anymore since VTK 8.2."
-					  "Please switch to the OpenGL2 backend instead, which is available since VTK 6.2."
-					  "Support of the deprecated backend will be dropped with PCL 1.13.")
-
 #If VTK version 7,8 or 9 use OpenGL2
-else()
+if(NOT (VTK_VERSION VERSION_LESS 7.0))
```

Comment:
```suggestion
if(VTK_VERSION VERSION_GREATER_EQUAL 7.0))
```

Reply:
In case of VTK 6.2/6.3 these Variables are now undefined. In case my research in #4065 was correct, you could define `OpenGL2` already for the older releases, so you can remove the whole `if`

Reply:
ok, i removed the whole if.